edge_computing/day_2_core_cnf_clusters/updating/update-completing-the-control-plane-only-update.adoc

```diff
@@ -0,0 +1,69 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="update-completing-the-update"]
+= Completing the Control Plane Only cluster update
+include::_attributes/common-attributes.adoc[]
+:context: completing-the-update
+
+toc::[]
+
+Follow these steps to perform the Control Plane Only cluster update and monitor the update through to completion.
```

Comment:
Consider re-writing:

Complete the following steps to update the control plane nodes.